distributed/tests/test_client.py

```diff
@@ -4355,6 +4355,7 @@ def test_normalize_collection_with_released_futures(c):
 
 
 @gen_cluster(client=True)
+@pytest.mark.xfail(reason="https://github.com/dask/distributed/issues/4404")
```

Comment:
It looks like we need to switch the order of `@pytest.mark.xfail` and `@gen_cluster` in order for the `xfail` to take affect

Reply:
Thanks James! Fixed below 😄